examples/utils/bundle-size-tests/src/sharedTree.ts

```diff
@@ -2,8 +2,8 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-import { SharedTreeFactory } from "@fluidframework/tree";
+import { SharedTree } from "@fluidframework/tree";
 
 export function apisToBundle() {
-	new SharedTreeFactory();
+	SharedTree.getFactory();
```

Comment:
SharedTreeFactory is @internal.

We were getting away with this before because tree still exposes untimmed to CJS (oops...  to fix separately.)